ros2topic/test/test_info.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from argparse import Namespace
+from contextlib import contextmanager
+from io import StringIO
+import sys
+
+from ros2topic.verb.info import InfoVerb
+
+NODE_NAME = 'bar'
+TOPIC_NAME = '/foo'
```

Comment:
Please use a topic name which is specific to this test, e.g. "test_ros2_topic_cli".

Reply:
Fixed.